core/trino-main/src/test/java/io/trino/sql/query/TestRowFilter.java

```diff
@@ -554,4 +562,24 @@ public void testRowFilterOnHiddenColumn()
                 .skippingTypesCheck()
                 .matches("SELECT BIGINT '25'");
     }
+
+    @Test
+    public void testRowFilterOnOptionalColumn() {
+        accessControl.reset();
+        accessControl.rowFilter(
+                new QualifiedObjectName(MOCK_CATALOG, "tiny", "nation_with_optional_column"),
+                USER,
+                new ViewExpression(USER, Optional.empty(), Optional.empty(), "optional is null"));
+
+        assertThatThrownBy(() -> assertions.query("INSERT INTO mock.tiny.nation_with_optional_column VALUES (101, 'POLAND', 0, 'No comment', 'some string')"))
```

Comment:
What is a default value? If it is `null` then it should pass.

Reply:
Could you clarify this?